manifests/integrations/http_check.pp

```diff
@@ -217,6 +219,8 @@
       'check_certificate_expiration' => $check_certificate_expiration,
       'days_warning'                 => $days_warning,
       'days_critical'                => $days_critical,
+      'check_hostname'                => $check_hostname,
```

Comment:
Can you align the rocket arrows on the right?

Reply:
Done 👍